GuessingGame.Tests/UnitTest1.cs

```diff
@@ -7,7 +7,7 @@ namespace GuessingGame.Tests;
 public class UnitTest1

```

Comment:
For production code you would name the unit test class something more meaningful for the use case. I usually name them after the class I am testing in your case it would be ServiceTests.